samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,46 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+Requirements
+************
+
+* A board with SPI slave, GPIO and BLE support
```

Comment:
> while I'd love to add even more 96Boards branding

I personally don't see it as "branding". This is a niche app, which won't work for 95% of boards. Someone may even wonder if there's a board for which it works, and can find it out. Yes, there're other ways to find it out, but the point of docs (README in our case) is to provide enough information, not make people "reverse engineer" it.

My overall concern is that this is not the first and not the last niche app, and not providing clear and useful info for each means that we target towards obscurity and esotericity (like someone with a board X may try 2, 3, 4 apps, none working for their board and silent about which boards it may work with).

YMMV